cmd/piper.go

```diff
@@ -123,6 +123,7 @@ func Execute() {
 	rootCmd.AddCommand(VaultRotateSecretIdCommand())
 	rootCmd.AddCommand(TransportRequestUploadCTSCommand())
 	rootCmd.AddCommand(IntegrationArtifactDeployCommand())
+	rootCmd.AddCommand(TransportRequestCreateSOLMANCommand())
```

Comment:
put the transportrequest commands together

Reply:
They are sorted chronologically 😄